packages/vue-cli-plugin-vuetify/generator/templates/v3/src/App.ts.vue

```diff
@@ -0,0 +1,11 @@
+<template>
```

Comment:
Should this have anything in it?

Reply:
I'm going to fill out the component templates today.